docs/fidoiot/fidoiot-introduction.md

```diff
@@ -0,0 +1,6 @@
+The SDO project is developing an implementation of the FIDO IOT device onboarding specification currently under development by the FIDO Alliance.  A [specification working draft](https://fidoalliance.org/specs/fidoiot/FIDO-IoT-spec-v1.0-wd-20200730.html) was released in August 2020.  
+
+The [PRI-FIDOIOT](https://github.com/secure-device-onboard/pri-fidoiot) component has been added to the LF Edge Secure Device Onboard project.  It is a reference implementation of the specification working draft, and also implements the service modules required to exercise the complete system: Device, Manufacturer, Rendezvous Server, and Owner.  See the [readme](https://github.com/secure-device-onboard/pri-fidoiot/blob/master/README.md) in the PRI-FIDOIOT project repository for details on the implementation and how to get it set up and running.  See the [release notes](https://github.com/secure-device-onboard/release/releases) for PRI-FIDOIOT 0.3 for a summary of features and capabilities implemented (or not) in the 0.3 release.
+
+The Client SDK - FIDOIOT component will be added to the SDO project in the near future.  It will be based on the current SDO Client SDK.  A [design document](fidoiot-client-sdk-design.md) outlining the changes required to support the new FIDO protocols for devices as described in the current draft has been added to the SDO documentation.  Comments are welcome and can be submitted via the SDO documentation github repository.
```

Comment:
1. in the near future (too wordy) --> soon.
2. github  --> GitHub.

Reply:
@DukeDavis12 Thanks for your comments.  I will be pushing an update shortly.